### ISSUE TITLE
Deal with wrong normilzations

### ISSUE THREAD

**Issue by pfefferniels** [pfefferniels/probstuecke-digital#124]
While generally text normalization using DTA's CAB service works fine, it introduces some mistakes, e.g. Schleiffer gets normalized to "Schluffer" and some more. Probably we should collect those and correct them after normalization again.

**Comment by pfefferniels** [comment]
further mistake in PS 20: Palmulen => Palmölen

**Comment by pfefferniels** [comment]
Not sure if relevant, but CAB seems to make use of an ["exception lexicon"](https://kaskade.dwds.de/~moocow/software/DTA-CAB/doc/html/DTA.CAB.WebServiceHowto.html#exlex)

**Comment by rettinghaus** [comment]
Could we report those errors for improving the language model?

**Comment by pfefferniels** [comment]
Yes absolutely – I'd suggest to collect as many as possible and report them in one go. Not sure where exactly to report though … (Bryan Jurish perhaps?)
